.github/workflows/auto-merge/automerge

```diff
@@ -119,14 +120,36 @@ def comment(number, content):
         print(r.json())
 
 
-def main():
-    number, sha, term = create()
-    if term:
-        sys.exit(0)
+def main(github_token: str, nvauto_token: str, owner: str, repo_name: str, head: str, base: str):
+    try:
+        # NOTE: github actions can not be trigger by default github-actions bot
+        # So use nvauto_token to trigger checks
+        number, sha, term = create(nvauto_token=nvauto_token, owner=owner, repo_name=repo_name, head=head, base=base)
+        if term:
+            sys.exit(0)
+
+        # skip pre-merge blossom CI w/ [skip ci] in PR title
+        comment(token=nvauto_token, owner=owner, repo_name=repo_name, number=number, content='build')
 
-    time.sleep(10)  # sleep, then comment
-    auto_merge(number, sha)
+        time.sleep(600)  # sleep 10 minutes, then try auto-merge
```

Comment:
Won't this seriously eat into our github action minutes?

Reply:
`github actions` is currently free to use in public repo. And we can easily turn this off or move it back to our jenkins if github change their mind :)
![image](https://user-images.githubusercontent.com/8086184/92290401-8832c800-ef46-11ea-8317-187783782e18.png)